lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -549,6 +549,9 @@ WasmBytecodeGenerator::EmitExpr(WasmOp op)
         info = EmitInfo(tempReg, WasmTypes::I32);
         }
         break;
+    case wbUnreachable:
+        m_writer.EmptyAsm(Js::OpCodeAsmJs::Unreachable_Void);
```

Comment:
@Cellule  right, sorry! w/o `info = EmitInfo(WasmTypes::Unreachable);` this `(select i32.const 0 i32.const 1 unreachable)` would probably make a verifier **unhappy**. i'm still getting a handle on a WASM verification process :-) im a bit more used to a fixed-point w/ data-flow verification model :-)